playbooks/roles/katello_provisioning/tasks/katello.yml

```diff
@@ -118,13 +118,6 @@
     --unlimited-hosts
   when: katello_provisioning_activation_key.stderr.find('not found') != -1
 
-- name: 'medium env should be in locs'
```

Comment:
Yea, this is OK.  The playbook is broken because we removed creation of installation media for Katello repos.  This fixes it, since they're attaching the kickstart repo ID instead later on.